src/views/OnboardingSelector.vue

```diff
@@ -111,7 +114,9 @@ export default class OnboardingSelector extends Vue {
     }
 
     private close() {
-        if (this.isSecondaryOnboarding) {
+        if (this.isSecondaryOnboarding
+            // In the case we don't have wallets while signing a message, we close the hub
+            && (this.originalRouteName !== RequestType.SIGN_MESSAGE || this.hasWallets)) {
```

Comment:
I don't understand this condition and the comment. The comment talks about _not_ having wallets, but the check is for _having_ wallets.